flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -307,13 +320,13 @@ public void testAlterTableAddColumn() {
     assertThat(schemaBefore.asStruct())
         .isEqualTo(
             new Schema(Types.NestedField.optional(1, "id", Types.LongType.get())).asStruct());
-    sql("ALTER TABLE tl ADD (dt STRING)");
+    sql("ALTER TABLE tl ADD (dt STRING COMMENT 'comment for dt')");
     Schema schemaAfter1 = table("tl").schema();
     assertThat(schemaAfter1.asStruct())
         .isEqualTo(
             new Schema(
                     Types.NestedField.optional(1, "id", Types.LongType.get()),
-                    Types.NestedField.optional(2, "dt", Types.StringType.get()))
+                    Types.NestedField.optional(2, "dt", Types.StringType.get(), "comment for dt"))
                 .asStruct());
     // Add multiple columns
     sql("ALTER TABLE tl ADD (col1 STRING, col2 BIGINT)");
```

Comment:
I would just change of the column here with comment and leave the line 310/323 not changed

Reply:
done.